src/Goteo/Library/Text.php

```diff
@@ -56,6 +56,28 @@ static public function sys($id, $args = null) {
         return static::lang($id, Config::get('lang'), $args);
     }
 
+    /**
+     * Compatibility text translation getter (instead of Lang::trans())
+     * @param  string $id key to translate (will be searched in all group translations)
+     * @param  [type] $lang [description]
+     * @return string     [description]
+     */
+    static public function getByLang ($id, $lang = null, $args = null) {
+        // Compatibilize with random number of arguments included
+        if(!is_array($args)) {
+            $args = func_get_args();
+            if (count($args) > 1) {
+                array_shift($args);
+            } else {
+                $args = array();
+            }
+        }
+
+        if (!$lang) $lang = Lang::current();
+
+        return static::lang($id, $lang, $args);
+    }
```

Comment:
This whole thing about `func_get_args` is to maintain compatibility with the old behaviour, we don't want to use that in the future. Any use of this function should be substituted directly by `Text::lang`.
So I'd suggest to remove this function